packages/protocol/contracts-0.8/stability/FeeCurrencyAdapter.sol

```diff
@@ -0,0 +1,146 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.8.7 <0.8.20;
+
+import "@openzeppelin/contracts8/access/Ownable.sol";
+import "@openzeppelin/contracts8/token/ERC20/IERC20.sol";
+
+import "../../contracts/common/CalledByVm.sol";
+import "../../contracts/common/Initializable.sol";
+import "../../contracts/common/interfaces/ICeloVersionedContract.sol";
+import "../../contracts/common/FixidityLib.sol";
+import "../../contracts/stability/interfaces/ISortedOracles.sol";
+import "./interfaces/IFeeCurrency.sol";
+import "./interfaces/IDecimals.sol";
+
+contract FeeCurrencyAdapter is Initializable, CalledByVm {
```

Comment:
Add `decimals()` and check if there's any other ERC20 method that would make sense. Think on a wallet trying to display an amount, a balance or something like that

Reply:
I added decimals and also totalSupply

Reply:
from my point of view it should return same deciamls as all other methods